docs/intro/install.rst

```diff
@@ -0,0 +1,28 @@
+.. _`intro-install`:
+
+==================
+Installation guide
+==================
+
+Installing web-poet
+===================
+
+web-poet is a library that runs on Python 3.6 and above.
+
+If you’re already familiar with installation of Python packages, you can install
+web-poet and its dependencies from PyPI with:
+
+::
+
+    pip install web-poet
+
+Things that are good to know
+============================
+
+web-poet is written in pure Python and depends on two key Python packages:
```

Comment:
I'm not sure we should be listing dependencies in docs. Arguments against it:

* we would need to keep it in sync with setup.py
* users may start to depend on specific behavior, e.g. ResponseData being an attr.s class, while this may change
* I'm not sure how is this information actionable for a user.

Mentioning in docs that selectors are powered by parsel is good, but I'm not sure about duplicating all the dependencies here.

Reply:
Yeah, that's was an idea I took from Scrapy's documentation when writing scrapy-poet's one. It's weird here since this project has such a small number of deps. I've just rewritten this section following your tips.